src/particles/plasma/PlasmaParticleContainer.H

```diff
@@ -21,8 +21,8 @@
 struct PlasmaIdx
 {
     enum {
-        x=0, y, z,          // position
-        w,                  // weight
+        x=0, y,             // position
+        w,                  // weight, this will be returned by pos(2)
```

Comment:
Let's **never** access `w` like this 🤕.

Reply:
Of course, I wanted to make debugging a potential future issue slightly easier.